libs/spark/src/Unstable_AvatarButton/Unstable_AvatarButton.stories.tsx

```diff
@@ -77,6 +86,10 @@ ChildrenInitialsSTP.args = { children: '(Initials)' };
 ChildrenInitialsSTP.decorators = [sparkThemeProvider];
 ChildrenInitialsSTP.storyName = 'children=(Initials) (STP)';
 
+export const ChildrenInitialsMedium: Story = Template.bind({});
+ChildrenInitialsMedium.args = { children: '(Initials)', size: 'medium' };
+ChildrenInitialsMedium.storyName = 'children=(Initials) (medium)';
```

Comment:
```suggestion
ChildrenInitialsMedium.storyName = 'children=(Initials) size=medium';
```
Follow storyName pattern